src/controller.coffee

```diff
@@ -2204,10 +2223,8 @@ Editor::showDropdown = ->
       div.innerHTML = el.display
       div.className = 'droplet-dropdown-item'
 
-      # Match fonts
-      div.style.fontFamily = @fontFamily
-      div.style.fontSize = @fontSize
       div.style.paddingLeft = helper.DROPDOWN_ARROW_WIDTH
+      div.style.paddingRight = DROPDOWN_SCROLLBAR_PADDING
```

Comment:
Would it make sense to set this to nonzero only if the total height of the elements exceeds the max-height of the scroller?